src/client/versionChecking.js

```diff
@@ -1,6 +1,9 @@
+const states = require('../states')
+
 module.exports = function (client, options) {
   client.on('disconnect', message => {
     if (!message.reason) { return }
+    if (client.state === states.PLAY | client.state === states.CONFIGURATION) { return }
```

Comment:
Can you add a comment here explaining why?

Reply:
Just documented it.

Reply:
why use bitwise or here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR

Reply:
Fixed, didn't realize that was bitwise or.